test/box-luatest/gh_7493_memtx_hash_idx_repeatable_read_violation_test.lua

```diff
@@ -0,0 +1,55 @@
+local server = require('test.luatest_helpers.server')
+local t = require('luatest')
+
+local g = t.group()
+
+g.before_all(function(cg)
+    cg.server = server:new{
+        alias   = 'dflt',
+        box_cfg = {memtx_use_mvcc_engine = true}
+    }
+    cg.server:start()
+end)
+
+g.after_all(function(cg)
+    cg.server:drop()
+end)
+
+g.before_each(function(cg)
+    cg.server:exec(function()
+        local s = box.schema.create_space('s')
+        s:create_index('pk', {type = 'HASH'})
+    end)
+end)
+
+g.after_each(function(cg)
+    cg.server:exec(function()
+        box.space.s:drop()
+    end)
+end)
+
+--[[
+Checks that commit of a tuple replacing a gap-inserted tuple from a concurrent
+transaction is correctly handled.
+]]
+g.test_gap_tracking_full_key_corner_cases = function(cg)
```

Comment:
Test name is stale.

Reply:
Fixed, sorry, doing a lot of copy-paste for tests lately.